userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 1
+---
+
+# Installing with GitOps
+
+## Environment setup
+
+To install profiles with GitOps you will need to have completed the following environment setup.
+
+ :white_check_mark: [Cluster](/docs/tutorial-basics/setup#kubernetes-cluster)
+
+ :white_check_mark: [Pctl binary](/docs/tutorial-basics/setup#pctl-the-profiles-cli)
+
+ :white_check_mark: [Profiles CRDs and Flux CRDs](/docs/tutorial-basics/setup#profiles-crds-and-flux-crds)
+
+ :white_check_mark: [GitHub repo](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux)
+
+ :white_check_mark: [GitHub token](/docs/tutorial-basics/setup#personal-access-token)
```

Comment:
optional sort of? To do "gitops" you don't have to automatic creating the PR

Reply:
i am doing the "no pr" route in another page